Week1/exercise/w1/index.js

```diff
@@ -1,30 +1,53 @@
 console.log('Hack your future, right?!');
 
-const header = document.querySelector('h1');
-console.log('header: ', header);
+const selectHeader = document.querySelector('h1');
+console.log('header: ', selectHeader);
 
 // EXERCISE:
 // select "change image" button (querySelector)
+const selectButtonToChangeImage = document.getElementById('btn-changeImage');
+
+// define changeImage() function
+const changeImage = () => {
+  // in this function:
+  //    get image element
+  const selectImage = document.querySelector('img');
+  //    get imageInput element
+  const selectImageInput = document.getElementById('imageInput');
```

Comment:
It would be best to select these elements outside of the function call, since they do not change